functions-and-operators/json-functions/json-functions-modify.md

```diff
@@ -0,0 +1,346 @@
+
+```sql
+SELECT JSON_MERGE_PRESERVE('{"a": 1, "b": 2}','{"a": 100}', '{"c": 300}');
+```
+
+```
++--------------------------------------------------------------------+
+| JSON_MERGE_PRESERVE('{"a": 1, "b": 2}','{"a": 100}', '{"c": 300}') |
++--------------------------------------------------------------------+
+| {"a": [1, 100], "b": 2, "c": 300}                                  |
++--------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [JSON_MERGE()](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-merge)
+
+> **Warning:**
+>
+> This function is deprecated.
+
+A deprecated alias for [`JSON_MERGE_PRESERVE()`](#json_merge_preserve).
+
+## [JSON_REMOVE()](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-remove)
+
+The `JSON_REMOVE(json_doc, path [,path] ...)` function removes `path` from a JSON document and returns the result.
+
+Examples:
+
+This example removes the `b` attribute from the JSON document.
+
+```sql
+SELECT JSON_REMOVE('{"a": 61, "b": 62, "c": 63}','$.b');
+```
+
+```
++--------------------------------------------------+
+| JSON_REMOVE('{"a": 61, "b": 62, "c": 63}','$.b') |
++--------------------------------------------------+
+| {"a": 61, "c": 63}                               |
++--------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+This example removes the `b` and `c` attributes from the JSON document.
+
+```sql
+SELECT JSON_REMOVE('{"a": 61, "b": 62, "c": 63}','$.b','$.c');
+```
+
+```
++--------------------------------------------------------+
+| JSON_REMOVE('{"a": 61, "b": 62, "c": 63}','$.b','$.c') |
++--------------------------------------------------------+
+| {"a": 61}                                              |
++--------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+## [JSON_REPLACE()](https://dev.mysql.com/doc/refman/8.0/en/json-modification-functions.html#function_json-replace)
+
+The `JSON_REPLACE(json_doc, path, value [, path, value] ...)` function replaces existing values in a JSON document and returns the result. If the specified path does not exist, it is not added to the result.
+
+This function takes arguments in pairs, where each pair is a `path` and a `value`.
+
+Examples:
+
+In the following example, you change the value at `$.b` from `62` to `42`.
+
+```sql
+SELECT JSON_REPLACE('{"a": 41, "b": 62}','$.b',42);
+```
+
+```
++---------------------------------------------+
+| JSON_REPLACE('{"a": 41, "b": 62}','$.b',42) |
++---------------------------------------------+
+| {"a": 41, "b": 42}                          |
++---------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+In the following example, you can change the value at `$.b` from `62` to `42`. In addition to this, you can replace the value at `$.c` with `43`. Because it does not exits in `json_doc`, the result is not changed.
```

Comment:
```suggestion
In the following example, you can change the value at `$.b` from `62` to `42`. In addition to this, you can replace the value at `$.c` with `43`. Because it does not exit in `json_doc`, the result is not changed.
```